qgis.desktop

```diff
@@ -2,11 +2,11 @@
 [Desktop Entry]
 Name=QGIS 2.18 LTR in Docker
 GenericName=QGIS 2.18 LTR in Docker
-Comment=GIS 2.18 LTR in Docker
-Exec=~/Documents/launch_scripts/docker_qgis.sh %U
+Comment=QGIS 2.18 LTR in Docker
+Exec=/home/norwin/bin/qgis2 %U
```

Comment:
here the user name is hardcoded, could that be changed?

Reply:
That is what this PR is primarily about:
In my environment (Gnome 3.32.2), the .desktop file is not recognized when relative paths or enironment variables are in the file.